src/altinn-app-frontend/src/components/summary/SummaryComponent.test.tsx

```diff
@@ -28,7 +28,7 @@ describe('SummaryComponent', () => {
             (t) =>
               ({
                 id: t,
-                type: t,
+                type: t === 'group' ? 'Group' : t,
```

Comment:
I dont think really gives any value - the test now just fixes the incorrectly cased implementation, and this would not affect any production code. 

I would suggest to just remove the `group` entry on line 22, and leave `t` here instead.

Reply:
To be fair, it introduces a new case in the test where `id` and `type` actually differs... :wink: Kidding aside, I agree - I'll remove the useless test.